File: solutions/go/isogram/3/isogram.go
// Package isogram returns true is the argument is an isogram else return false
package isogram

import "strings"

// IsIsogram determines is is argument is an isogram
func IsIsogram(word string) bool {
	var cc [26]int
	for _, c := range []rune(strings.ToLower(word)) {
    if c >= 'a' && c <='z' {
		  cc[int(c)-int('a')]++
		  if cc[int(c)-int('a')] > 1 {
			  return false
		  }
    }
	}
	return true
}

File: solutions/c/isogram/2/src/isogram.c
#include "isogram.h"
#include "ctype.h"
#include "stdio.h"

bool is_isogram(const char phrase[])
{
	int lc[26] = {0};
	for (int i = 0; phrase[i]!='\0'; i++) 
	{
		//printf("%d, %c", i, phrase[i]);
		if (isalpha(phrase[i]))
		{
			lc[(int)(tolower(phrase[i]))-(int)('a')]++;
			//printf("%d",lc[(int)(tolower(phrase[i]))-(int)('a')]);
			if (lc[(int)(tolower(phrase[i]))-(int)('a')]>1) return false;
		}

	}
	return true;
}
